src/connector/src/sink/snowflake_connector.rs

```diff
@@ -221,25 +224,49 @@ impl SnowflakeS3Client {
         // create the brand new s3 client used to sink files to s3
         let s3_client = S3Client::new(&config);
 
+        // just use default here
+        let config = ObjectStoreConfig::default();
```

Comment:
plus, do we need special parameters for the configurations?

Reply:
Not necessarily, but it would be better to allow users to pass parameters to the S3 client (OpenDAL) if possible. Just for illustration, any parameters start with `opendal.param_name = param_val` with be passed to OpenDAL.

Reply:
> but it would be better to allow users to pass parameters to the S3 client (OpenDAL) if possible.

agree, I could put the support of extra configurations for s3 client in subsequent pr.